graphql/models/Meta.graphql

```diff
@@ -0,0 +1,15 @@
+type Query{
+    metes: [Meta!]! @paginate(defaultCount:10) 
+    meta(id:ID @eq):Meta @find
+}
+
+type Meta{
+    id: ID!
+    key:String!
+    value:String!
+    metable: Metable! @morphTo
+}
+union Metable = Center|Meta
```

Comment:
those union types are incorrect, `metable` should be one of the models that have an image like `Center` or `Place`, so the correct type should be 
```
union Metable = Center|Place|Service|User|City|Governerate
```